### PR TITLE
add pkg: asciinema

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: pkgs/tools/misc/asciinema/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, goPackages }:
+
+goPackages.buildGoPackage rec {
+  name = "asciinema-${version}";
+  version = "v1.1.1";
```
Comment (jagajaga) [inline pkgs/tools/misc/asciinema/default.nix]: Remove `v`.

**Comment by jagajaga** [comment]
Rename commit as it said in `CONTRIBUTING.md`

**Comment by jagajaga** [comment]
@garbas hey, once again, why are you merging before fixing these little things? We need our repo to be more clean.

**Comment by spwhitt** [comment]
@jagajaga garbas made the changes you asked for in the commit he pushed. Check the diff.

**Comment by jagajaga** [comment]
Thank you :)
But we've already solved that problem.

**Pull request closed without merging**